versions.tf

```diff
@@ -0,0 +1,15 @@
+##############################################################################
+# Terraform Providers
+##############################################################################
+
+terraform {
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = ">=1.52.1"
```

Comment:
Is there a particular reason why we need to update the version from `>= 1.51.0` to `>=1.52.1` ? Perhaps some bug fix or feature that came in this version? Or can we just put back the content of what was in `version.tf` ?

Reply:
this was a request by eviden